### PR TITLE
Split config into environment-specific files

### PR DESCRIPTION
**Pull request by suzannehamilton**
This will make it easier to add different CORS configuration to integration vs staging and production.

It will also make it easier to set up a development environment, because the local database connection config is in the dev-specific config file.

https://national-archives.atlassian.net/browse/TDR-113

### CODE REVIEW

File: src/main/resources/application.base.conf [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  url = ${AUTH_URL}
+}
+frontend = {
+  url = ${FRONTEND_URL}
+}
+
+source = {
+  uuid = uk.gov.nationalarchives.tdr.api.service.RandomUUID
+}
+
```
Comment (TomJKing) [inline src/main/resources/application.base.conf:11]: Extra line break

Comment (suzannehamilton) [reply src/main/resources/application.base.conf:12]: Fixed.

**Comment by suzannehamilton** [comment]
The Readme's still up-to-date, because we didn't get around to adding the extra environment variables to it and now they're not necessary. Eventual consistency. 🙂

**Comment by suzannehamilton** [comment]
I'm not going to merge this yet because https://github.com/nationalarchives/tdr-terraform-environments/pull/57 needs to be merged and deployed first.

**Pull request merged**